src/commands/install-docker.yml

```diff
@@ -76,16 +76,18 @@ steps:
         DOCKER_BINARY_URL="https://download.docker.com/$PLATFORM/static/stable/x86_64/docker-$DOCKER_VERSION_NUMBER.tgz"
 
         # download binary tarball
-        curl --output docker.tgz \
+        DOWNLOAD_DIR="$(mktemp --directory)"
```

Comment:
Could you try `-d` instead? It looks like `--directory` doesn't work on Mac and Alpine images.
(See: https://circleci.com/gh/CircleCI-Public/docker-orb/2443)

Reply:
Fixed